net/sslh/src/opnsense/service/conf/actions.d/actions_sslh.conf

```diff
@@ -0,0 +1,31 @@
+################################################################################
+# Service Actions                                                              #
+################################################################################
+
+[status]
+command:/usr/local/etc/rc.d/sslh onestatus || exit 0
+parameters:
+type:script_output
+message: sslh: requesting status
+description:
+
+[start]
+command:/usr/local/etc/rc.d/sslh onestart || exit 0
```

Comment:
"one"start etc. suggests you don't have rc.conf wired up and you should do that via templates. it makes autostart on boot possible...